src/components/CareerFramework.vue

```diff
@@ -27,16 +27,20 @@
                 :class="level.level"
                 class="level"
               >
-                <b>{{ level.level }}</b
-                ><br />
-                <ul v-for="criteria in level.criteria" :key="criteria">
-                  <li>
-                    {{ criteria }}
-                  </li>
-                </ul>
+                <div v-if="level.criteria.length">
```

Comment:
```suggestion
                <div v-if="level.criteria.length">
```
What happened if you put the `v-if` on the surrounding `td` element?

Reply:
I think then it won't render the TD which will probably give us problems in a situation where there is an empty column somewhere in the middle of the row 🤔

Reply:
yeah makes sense